libs/openFrameworksCompiled/project/linuxarmv6l/config.linuxarmv6l.default.mk

```diff
@@ -205,10 +208,8 @@ endif
 ################################################################################
 
 # Broadcom hardware interface library
-PLATFORM_HEADER_SEARCH_PATHS += $(RPI_ROOT)/opt/vc/include
-PLATFORM_HEADER_SEARCH_PATHS += $(RPI_ROOT)/opt/vc/include/IL
-PLATFORM_HEADER_SEARCH_PATHS += $(RPI_ROOT)/opt/vc/include/interface/vcos/pthreads
-PLATFORM_HEADER_SEARCH_PATHS += $(RPI_ROOT)/opt/vc/include/interface/vmcs_host/linux
+PLATFORM_HEADER_SEARCH_PATHS += $(RPI_ROOT)/userland/host_applications/linux/libs/bcm_host/include
+PLATFORM_HEADER_SEARCH_PATHS += $(RPI_ROOT)/userland
```

Comment:
Will this work for people working with the device itself ( as opposed to building in the CI )? 
ie: does userland exist for Pi device users?

Reply:
Good catch. maybe I should put back the includes with /opt/vc in armv6l
easy to do

Reply:
and enclose this in ifdef CROSS_COMPILE